tests/integration-tests/test_runner.py

```diff
@@ -244,6 +251,7 @@ def _get_pytest_args(args, regions, log_file, out_dir):
     pytest_args.extend(["--output-dir", "{0}/{1}".format(args.output_dir, out_dir)])
     pytest_args.extend(["--key-name", args.key_name])
     pytest_args.extend(["--key-path", args.key_path])
+    pytest_args.append("--stackname-suffix={0}".format(args.stackname_suffix))
```

Comment:
minor: just for consistency we can use the extend and an array as in the previous line

Reply:
If you extend the array like in the previous one and the suffix contains a -, it will be seen as a parameter